20FinalProject/index.html

```diff
@@ -1,10 +1,181 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Kira Villarreal</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Cherry+Cream+Soda" rel="stylesheet">
+  <link rel="stylesheet" href= "https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+
+  <link rel="stylesheet" href="https://s3-us-west-2.amazonaws.com/s.cdpn.io/185649/owl.carousel.min.css" />
+  <link rel="stylesheet" href="https://s3-us-west-2.amazonaws.com/s.cdpn.io/185649/owl.theme.default.min.css" />
+
+  <!--<link rel="stylesheet" href="http://www.owlcarousel.owlgraphic.com/assets/owlcarousel/assets/owl.carousel.min.css">
+  <link rel="stylesheet" href="http://www.owlcarousel.owlgraphic.com/assets/owlcarousel/assets/owl.theme.default.min.css" />-->
+
+
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <div class="page">
+      <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
+        <div class="container-fluid">
+          <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#example-navbar-collapse">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+            <a class="navbar-brand" href="#">Kira Villarreal</a>
+          </div>
+
+          <!-- Collect the nav links for toggling -->
+          <div class="collapse navbar-collapse" id="example-navbar-collapse">
+            <ul class="nav navbar-nav">
+              <li>
+                <a href="#">Home</a>
```

Comment:
If you add an id to the `header` element, you can target that in this "Home" link. It's an oddball out right now since it doesn't animate the return to the top of the page.